source/Government.cpp

```diff
@@ -169,37 +169,31 @@ const Color &Government::GetColor() const
 
 
 // Get the government's tribute-related hails
-const Phrase *Government::TributeIgnoredHail() const
+const Phrase *Government::GetTributeHail(int condition)
 {
-	return tributeIgnoredHail;
-}
-
-
-
-const Phrase *Government::TributePresentHail() const
-{
-	return tributePresentHail;
-}
-
-
-
-const Phrase *Government::TributeInProgressHail() const
-{
-	return tributeInProgressHail;
-}
-
-
-
-const Phrase *Government::TributeCombatHail() const
-{
-	return tributeCombatHail;
-}
-
-
-
-const Phrase *Government::TributeSurrenderedHail() const
-{
-	return tributeSurrenderedHail;
+	switch(condition)
```

Comment:
fun fact: this would be the only use of `switch` in all of es.
I was thinking of something like what is used for colors:
https://github.com/endless-sky/endless-sky/blob/89259410371ab124580751c98fd1e332b35e6a58/source/Radar.cpp#L89